Chap_Revisions.tex

```diff
@@ -537,21 +537,30 @@ \section{Version 4.0: Nov. 2020}
 
     \item Server \acp{API}
     \begin{compactitemize}
-    \item \refapi{PMIx_server_generate_locality_string}
-    \item \refapi{PMIx_Register_attributes}
-    \item \refapi{PMIx_server_define_process_set}, \refapi{PMIx_server_delete_process_set}
-    \item \refapi{pmix_server_grp_fn_t}, \refapi{pmix_server_fabric_fn_t}
-    \item \refapi{pmix_server_client_connected2_fn_t}
-    \item \refapi{PMIx_server_generate_cpuset_string}
-    \item \refapi{PMIx_server_register_resources}, \refapi{PMIx_server_deregister_resources}
+        \item \refapi{PMIx_server_generate_locality_string}
+        \item \refapi{PMIx_Register_attributes}
+        \item \refapi{PMIx_server_define_process_set}, \refapi{PMIx_server_delete_process_set}
+        \item \refapi{pmix_server_grp_fn_t}, \refapi{pmix_server_fabric_fn_t}
+        \item \refapi{pmix_server_client_connected2_fn_t}
+        \item \refapi{PMIx_server_generate_cpuset_string}
+        \item \refapi{PMIx_server_register_resources}, \refapi{PMIx_server_deregister_resources}
     \end{compactitemize}
 
     \item Tool \acp{API}
     \begin{compactitemize}
-    \item \refapi{PMIx_tool_disconnect}
-    \item \refapi{PMIx_tool_set_server}
-    \item \refapi{PMIx_tool_attach_to_server}
-    \item \refapi{PMIx_tool_get_servers}
+        \item \refapi{PMIx_tool_disconnect}
+        \item \refapi{PMIx_tool_set_server}
+        \item \refapi{PMIx_tool_attach_to_server}
+        \item \refapi{PMIx_tool_get_servers}
+    \end{compactitemize}
+
+    \item Macros
+    \begin{compactitemize}
+        \item \refmacro{PMIX_INFO_LIST_START}
+        \item \refmacro{PMIX_INFO_LIST_ADD}
+        \item \refmacro{PMIX_INFO_LIST_XFER}
+        \item \refmacro{PMIX_INFO_LIST_CONVERT}
+        \item \refmacro{PMIX_INFO_LIST_RELEASE}
```

Comment:
The macros should be moved to the `Added Macros` section with `PMIX_CHECK_RESERVED_KEY`